/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/mmengine/utils/dl_utils/setup_env.py:56: UserWarning: Setting MKL_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed.
  warnings.warn(
/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/utils/data/dataloader.py:487: UserWarning: This DataLoader will create 2 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Traceback (most recent call last):
  File "tools/train.py", line 180, in <module>
    main()
  File "tools/train.py", line 158, in main
    runner.train()
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/mmengine/runner/runner.py", line 1723, in train
    self._init_model_weights()
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/mmengine/runner/runner.py", line 906, in _init_model_weights
    model.init_weights()
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/mmengine/model/base_module.py", line 136, in init_weights
    m.init_weights()
  File "/home/nr4325/Desktop/Pose4/mmpose/mmpose/models/backbones/wtpose.py", line 1271, in init_weights
    table_current = self.state_dict()[table_key]
KeyError: 'backbone.stages.0.blocks.0.attn.w_msa.relative_position_bias_table'
/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launch.py:178: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use_env is set by default in torchrun.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 2371180) of binary: /home/nr4325/miniconda3/envs/pose4/bin/python
Traceback (most recent call last):
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launch.py", line 193, in <module>
    main()
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launch.py", line 189, in main
    launch(args)
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launch.py", line 174, in launch
    run(args)
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/run.py", line 715, in run
    elastic_launch(
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 131, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/nr4325/miniconda3/envs/pose4/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 245, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
tools/train.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-01-26_16:51:21
  host      : skl-a-48.rc.rit.edu
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 2371180)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
